PracticeGit/intlist.c

```diff
@@ -54,9 +55,9 @@ int RemoveAt (struct node* head, int index) {
    return SUCCESS;
 }
 

```

Comment:
Try to handle edge cases :
Edge Case in Insert: Handle cases where the index is 0.
Edge Case in RemoveAt: Handle the case where index is 0.